src/app/users/getManageConsoleRoles.js

```diff
@@ -0,0 +1,69 @@
+const config = require('../../infrastructure/config');
+const { sendResult } = require('../../infrastructure/utils');
+const { getUserDetails} = require('./utils');
+const { getServiceById } = require('../../infrastructure/applications')
+const { listRolesOfService, getSingleUserService, getSingleInvitationService, updateUserService } = require('../../infrastructure/access');
+
+const getSingleServiceForUser = async (userId, organisationId, serviceId, correlationId) => {
+  const userService = userId.startsWith('inv-') ? await getSingleInvitationService(userId.substr(4), serviceId, organisationId, correlationId) : await getSingleUserService(userId, serviceId, organisationId, correlationId);
+  const application = await getServiceById(serviceId, correlationId);
+
+  return {
+    id: userService.serviceId,
+    roles: userService.roles,
+    name: application.name
+  }
+};
+
+const addOrChangeManageConsoleServiceTitle = (userManageRoles, manageConsoleRoleIds) => {

```

Comment:
You might want to consider something like the following:
```
const addOrChangeManageConsoleServiceTitle = (userManageRoles, manageConsoleRoleIds) => {
  const userManageConsoleRoleIds = userManageRoles.map(role => role.id);
  return manageConsoleRoleIds.some(roleId => userManageConsoleRoleIds.includes(roleId));
}
```

Also it's better to use `const` where possible over `let`.

Reply:
Thanks Lea. With this one, I would like to return a boolean, not an array. I think this should do the trick?? 

`const addOrChangeManageConsoleServiceTitle = (userManageRoles, manageConsoleRoleIds) => {
  return userManageRoles.roles
    .map(role => role.id)
    .some(roleId => manageConsoleRoleIds.includes(roleId));
}`

Reply:
Hi @alexplattdfe you should be able to achieve the above without .map since you can check `role.id` instead of mapping an array of roles to an array of roleIds within the `.some` predicate.

Reply:
Thanks for the feedback, i have now made these changes.